contrib/nghttpx.service.in

```diff
@@ -1,10 +1,18 @@
 [Unit]
 Description=HTTP/2 proxy
+Documentation=man:nghttpx
 After=network.target
 
 [Service]
-Type=forking
-ExecStart=@bindir@/nghttpx --conf=/etc/nghttpx/nghttpx.conf --pid-file=/run/nghttpx.pid --daemon
+Type=notify
+ExecStart=@bindir@/nghttpx --conf=/etc/nghttpx/nghttpx.conf
+ExecReload=/bin/kill --signal USR2 $MAINPID
```

Comment:
Usually, reload means "force reloading configuration".  If yes, SIGHUP is better choice.  Actually, SIGUSR2 does not cause nghttpx to shut down itself.